content/coronavirus_education_page.yml

```diff
@@ -15,36 +15,70 @@ content:
           list:
             - label: Supporting your children's education during coronavirus
               url: /guidance/supporting-your-childrens-education-during-coronavirus-covid-19
+            - label: Online education resources
+              url: /government/publications/coronavirus-covid-19-online-education-resources
+            - label: Remote education during coronavirus
+              url: /guidance/remote-education-during-coronavirus-covid-19
     - title: Pupil wellbeing and safety
       sub_sections:
         - title:
           list:
             - label: Supporting children and young people’s mental health and wellbeing
               url: /government/publications/covid-19-guidance-on-supporting-children-and-young-peoples-mental-health-and-wellbeing
+            - label: Free school meals
+              url: /government/publications/covid-19-free-school-meals-guidance
+            - label: Implementing social distancing in education and childcare settings
+              url: /government/publications/coronavirus-covid-19-implementing-social-distancing-in-education-and-childcare-settings
+            - label: Guidance on isolation for residential educational settings
+              url: /government/publications/coronavirus-covid-19-guidance-on-isolation-for-residential-educational-settings
+            - label: Supporting vulnerable children
+              url: /government/publications/coronavirus-covid-19-guidance-on-vulnerable-children-and-young-people
+            - label: SEND risk assessment guidance
+              url: /government/publications/coronavirus-covid-19-send-risk-assessment-guidance
     - title: Closures, exams and managing a school or early years setting
       sub_sections:
         - title:
           list:
             - label: Schools opening for children of key workers and vulnerable children
               url: /government/publications/coronavirus-covid-19-maintaining-educational-provision
+            - label: Guidance for schools and local authorities on closures
+              url: /government/publications/covid-19-school-closures
+            - label: Early years and childcare closures
+              url: /government/publications/coronavirus-covid-19-early-years-and-childcare-closures
+            - label: Guidance on early years education
+              url: /government/publications/coronavirus-covid-19-early-years-and-childcare-closures
```

Comment:
This URL appears to be a duplicate of the one above it, is that intended?

Reply:
That's what's in the design.  I'll flag to Alan when I hand over.  Thanks Andy!